apps/cde-ui/src/app/components/visual-card/visual-card.component.html

```diff
@@ -1,5 +1,11 @@
-@for (card of cardData(); track card.label) {
-  <a [routerLink]="card.route" class="visual-card cta-flat" rel="noopener noreferrer">
+@for (card of cardData(); track card.label; let i = $index) {
+  <a
+    [routerLink]="card.route"
+    class="visual-card cta-flat"
+    rel="noopener noreferrer"
+    [hraFeature]="featureNames()[i]"
```

Comment:
Change to `"card.label | slugify"` (make sure to pull from base branch to get `slugify` pipe)